schema.sql

```diff
@@ -31,3 +31,13 @@ CREATE TABLE label (
   title VARCHAR(30),
   color VARCHAR(20)
 );
+
+CREATE TABLE music-album (
```

Comment:
I just tested this table name in SQL and it throws an error, looks like the name must be in the camel case.
```suggestion
CREATE TABLE musicAlbum (
```